src/store/imageBuilderApi.ts

```diff
@@ -1,5 +1,13 @@
 import { emptyImageBuilderApi as api } from "./emptyImageBuilderApi";
+import { fetchBaseQuery } from '@reduxjs/toolkit/query'
+
 const injectedRtkApi = api.injectEndpoints({
+  baseQuery: fetchBaseQuery({paramsSerializer: (params) => {
```

Comment:
Can you try to write this in an other way ? Or maybe comment what's happening, because it's quite packed rn.